routes/login.ts

```diff
@@ -21,38 +16,38 @@ sgMail.setApiKey(process.env.SENDGRID_API as string);
 const client = new OAuth2Client(process.env.CLIENTID);
 const router = Router();
 
-router.get("/", async (req:any, res:any) => {
+router.get("/", async (req: any, res: any) => {
   if (req.session.user) {
-    const user = await getProfileById(Number(req.session.user));
+    const user = await getProfileById(req.session.user);
```

Comment:
wouldnt this need to be the uuid? Im pretty sure the req.session.user is the users id